iiwa_ros_java/src/de/tum/in/camp/kuka/ros/iiwaPublisher.java

```diff
@@ -36,6 +36,7 @@
 // import com.kuka.generated.ioAccess.MediaFlangeIOGroup; // MEDIAFLANGEIO
 import com.kuka.roboticsAPI.deviceModel.LBR;
 import com.kuka.roboticsAPI.geometricModel.ObjectFrame;
+import com.kuka.roboticsAPI.ioModel.AbstractIOGroup;

```

Comment:
Is AbstractIoGroup available also without the media flange software module?

Reply:
It seems so. I tested copying the code a new project with the flange set to `none` and it didn't show any errors.